main/modes/mode_nvs_manager.c

```diff
@@ -0,0 +1,3467 @@
+/*
+ * mode_nvs_manager.c
+ *
+ *  Created on: 3 Dec 2022
+ *      Author: bryce
+ */
+
+/*==============================================================================
+ * Includes
+ *============================================================================*/
+
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <limits.h>
+#include <math.h>
+
+//#include <esp32/rom/crc.h>
+
+#include "bresenham.h"
+#include "display.h"
+#include "embeddednf.h"
+#include "embeddedout.h"
+#include "esp_heap_caps.h"
+#include "esp_timer.h"
+#include "espNowUtils.h"
+#include "led_util.h"
+#include "linked_list.h"
+#include "meleeMenu.h"
+#include "mode_main_menu.h"
+#include "mode_test.h"
+#include "nvs_manager.h"
+#include "p2pConnection.h"
+#include "settingsManager.h"
+#ifdef NVS_MANAGER_TOUCH
+#include "touch_sensor.h"
+#endif
+#if defined(EMU)
+#include "emu_main.h"
```

Comment:
What's this needed for?

Reply:
I had been using the touchpad as buttons at one point before replacing those controls with other styles of controls, such as more menus. Left this in but uncompiled in case I ended up needing more buttons again, so I wouldn't need to recreate the variables, functions, and assignments. I can remove them if you prefer.

Reply:
Not the touchpad part, the `emu_main.h` part

Reply:
https://github.com/AEFeinstein/Super-2023-Swadge-FW/pull/378#discussion_r1051529565
Posted a comment on the relevant lines